index.js

```diff
@@ -1,37 +1,41 @@
 const { createLogger, format, transports } = require('winston');
 const expressWinston = require('express-winston');
+const { simple, errors, ecs } = require('./format');
```

Comment:
There's no errors object in the index, though.

Reply:
yep, i just removed it and didn't update yet, good catch